src/domain/messages/helpers/message-verifier.helper.ts

```diff
@@ -94,13 +101,15 @@ export class MessageVerifierHelper {
     safe: Safe;
     expectedHash: `0x${string}`;
     message: string | Record<string, unknown>;
+    source: LogSource;
   }): void {
     const calculatedHash = this.calculateMessageHash(args);
 
     if (calculatedHash !== args.expectedHash) {
       this.logMismatchMessageHash({
         ...args,
         messageHash: args.expectedHash,
+        source: args.source,
```

Comment:
As we are spreading `args` already, we don't need this.
```suggestion
```

Reply:
Adjusted in [d4919b8](https://github.com/safe-global/safe-client-gateway/pull/2433/commits/d4919b8c4ef2b7bf24ca01c381082416a711bf0f)